### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by aksenovaa**
Complete Russian translation of the manual

### CODE REVIEW

**Comment by linawolf** [comment]
@aksenovaa it is very nice you translated the whole tutorial, however are you also willing to maintain changes in the future? We have had the problems with translations before: They are made in a one time effort and then not maintained. 

As we have no one in the documentation team who speaks russian we can however not maintain this in the long term. If you would like to also offer maintanance please come to the TYPO3 Slack, channel `#typo3-documentation` and have a chat about how we can ensure long term updates and maintanance.

File: Documentation/Localization.ru_RU/About.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+.. include:: /Includes.rst.txt
+
+.. this file is linked from Index.rst but not included in the menu
+
+.. _about:
+.. _about-this-document:
+
+===================
+About This Tutorial
```
Comment (linawolf) [inline Documentation/Localization.ru_RU/About.rst:11]: This is english

File: Documentation/Localization.ru_RU/About.rst [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+===================
+
+Этот документ знакомит новых пользователей с TYPO3, ее основными функциями и даёт общее представление о настройке и администрированию CMS.
+
+Upon completing this tutorial, you should have a better understanding about how the CMS is installed, how the backend is administered and how you begin creating templates. По завершению знакомства с этим руководством, вы научитесь устанавливать CMS, получите представление об администрировании системы из интерфейса управления (backend) и создании шаблонов страниц сайта.
```
Comment (linawolf) [inline Documentation/Localization.ru_RU/About.rst:16]: Here is also english...

**Comment by aksenovaa** [comment]
> @aksenovaa it is very nice you translated the whole tutorial, however are you also willing to maintain changes in the future? We have had the problems with translations before: They are made in a one time effort and then not maintained.
> 
> As we have no one in the documentation team who speaks russian we can however not maintain this in the long term. If you would like to also offer maintanance please come to the TYPO3 Slack, channel `#typo3-documentation` and have a chat about how we can ensure long term updates and maintanance.

I've been working with TYPO3 for a long time, since 2006, and all the translations of the TYPO3 interface itself are my work.
I have also been translating manuals into Russian. The latest translations of press releases seem to be for TYPO3 10.
Back then I translated the manuals in LibreOffice. It was very inconvenient.
A lot has changed since then, there is some technology for translators and documentation.
Let's try to work on it. 

Translated with www.DeepL.com/Translator (free version)

**Comment by linawolf** [comment]
My problem is also that I cannot read or speak a single word in Russian, therefore I can do no review beyond syntax or untranslated sentences. @brotkrueml (Co-Lead) had some Russian in school but he is on vacation for the next 3 weeks.

**Comment by aksenovaa** [comment]
I think you don't need to know all languages ))))) TYPO3 is represented all over the world.
The way I see it, your job here is to make sure that technically everything is correct.

**Review by aksenovaa — COMMENTED** [review]
Deleted the translated text that was left inadvertently.

**Comment by linawolf** [comment]
Well all contents in the official manuals get reviewed by at least 2 ppl and the reviewer should ideally understand what the change is about. Also I am not yet sure about the technical side. There have been so many changes to the rendering pipeline since we last had a translated manual. And @marble  is also on vacation. And so will I starting from Tuesday. So please be patient with us

**Comment by aksenovaa** [comment]
There's no rush, especially not in official documentation. I understand it perfectly well ) 
I've worked in large companies, and I know what it's like to approve documents.

I plan to do documentation for editors as well.

**Comment by nhovratov** [comment]
Hello @aksenovaa, thank you very much for the russian translation! It looks really good :+1: We just have to streamline the rst syntax a bit here and there.

File: Documentation/Localization.ru_RU/Troubleshooting/Index.rst [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header h3
+
+            .. rubric:: :ref:`TYPO3 <troubleshooting_typo3>`
+
+         .. container:: card-body
+
+            *  :ref:`Reset Backend Administrative User Passwords<backend-admin-password>`
```
Comment (linawolf) [inline Documentation/Localization.ru_RU/Troubleshooting/Index.rst:24]: This page still needs some translations

File: Documentation/Localization.ru_RU/NextSteps/Index.rst [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+.. _next-steps:
+
+==============================
+Дальнейшие шаги и дополнительная литература
+==============================
+
+После установки TYPO3 можно приступать к разработке внешнего вида сайта и созданию страниц и содержимого внутри CMS.
+
+:doc:`Building The Sites Structure And Adding Content <t3editors:Index>`
```
Comment (linawolf) [inline Documentation/Localization.ru_RU/NextSteps/Index.rst:13]: Please translate this headline

File: Documentation/Localization.ru_RU/Settings.cfg [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+[intersphinx_mapping]
+
+
+t3start11        = https://docs.typo3.org/m/typo3/tutorial-getting-started/11.5/ru-ru/
+
+# Official TYPO3 manuals
+# h2document     = https://docs.typo3.org/m/typo3/docs-how-to-document/main/ru-ru/
+# t3cheatsheets  = https://docs.typo3.org/m/typo3/docs-cheatsheets/main/ru-ru/
+# t3contribute   = https://docs.typo3.org/m/typo3/guide-contributionworkflow/main/ru-ru/
+t3coreapi      = https://docs.typo3.org/m/typo3/reference-coreapi/main/ru-ru/
```
Comment (linawolf) [inline Documentation/Localization.ru_RU/Settings.cfg:35]: Linking to the Core API in russian will not work as we do not have a russian translation and as you agreed never will have. I would suggest to link to english instead

**Comment by linawolf** [comment]
There are also still a number of Syntax issues but we can fix them in a follow up. this is great work! Rendering it locally looks good

**Review by linawolf — APPROVED** [review]

**Comment by dmitryd** [comment]
You can also ping me for translations or checks. I am a native russian.

**Pull request merged**